instance.go

```diff
@@ -380,3 +391,15 @@ func (c *Client) SetInstanceFirewall(id, firewallID string) (*SimpleResponse, er
 	response, err := c.DecodeSimpleResponse(resp)
 	return response, err
 }
+
+// InitInstanceVNC gets the VNC details for an instance
+func (c *Client) InitInstanceVNC(id string) (*InstanceVNCDetails, error) {
+	resp, err := c.SendPutRequest(fmt.Sprintf("/v2/instances/%s/vnc", id), "")
+	if err != nil {
+		return nil, decodeError(err)
+	}
```

Comment:
did you tested it locally without mocking the API?
I don't think it will work rather it would throw that instance not found in this region error i assume